src/core/uuid/uuid.cc

```diff
@@ -1,3 +1,4 @@
+#include <array>
```

Comment:
Can you add this include alongside the system includes below and add the usual `std::array` comment?